src/components/Gradebook/gradebook.scss

```diff
@@ -27,6 +27,10 @@
     @media only screen and (min-width: 1200px) {
         width: 1024px;
     }
+
+    .search-help-text {
+        margin-left: 20px;
```

Comment:
UX/UI request: search help text should be aligned with the label in the SearchField